.gitignore

```diff
@@ -40,6 +40,7 @@ build/
 src/generated/
 .swagger-codegen/
 .swagger-codegen-ignore
+pom.xml
```

Comment:
The new `swagger-codegen` generates this file in the project root. I couldn't find a toggle to disable the behavior.

Reply:
Yes, it is a known bug